### PR TITLE
make a bunch of tests compatible with django 1.8

### PR DESCRIPTION
**Pull request by nickpell**
@emord cc: @snopoke

### CODE REVIEW

**Comment by emord** [comment]
:+1: on :white_check_mark:

File: corehq/blobs/tests/test_migrate.py [hunk 1]
```diff
@@ -29,6 +29,7 @@
 class BaseMigrationTest(TestCase):
```
Comment (millerdev) [inline corehq/blobs/tests/test_migrate.py]: Would `TestMigrateBackend` (lower in this file) benefit from these same super calls too? If not, can you explain why or point me to some documentation that explains why?

Comment (nickpell) [inline corehq/blobs/tests/test_migrate.py]: ```TestMigrateBackend``` would probably benefit too.  I just decided to stop at some arbitrary point for the day.  I'll make sure to get this one in the next batch.

**Comment by nickpell** [comment]
going to break up this PR since tests in ```custom/``` aren't behaving well

**Pull request closed without merging**